src/components/OSMElementData/OSMElementData.jsx

```diff
@@ -0,0 +1,186 @@
+import { Fragment, useState, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import { FormattedMessage, injectIntl }
+       from 'react-intl'
+import _map from 'lodash/map'
+import _compact from 'lodash/compact'
+import _isEmpty from 'lodash/isEmpty'
+import _sortBy from 'lodash/sortBy'
+import _find from 'lodash/find'
+import _get from 'lodash/get'
+import _flatten from 'lodash/flatten'
+import AsMappableTask
+       from '../../interactions/Task/AsMappableTask'
+import AsIdentifiableFeature
+       from '../../interactions/TaskFeature/AsIdentifiableFeature'
+import Dropdown from '../Dropdown/Dropdown'
+import SvgSymbol from '../SvgSymbol/SvgSymbol'
+import BusySpinner from '../BusySpinner/BusySpinner'
+import messages from './Messages'
+import './OSMElementData.scss'
+
+const OSM_SERVER = window.env.REACT_APP_OSM_SERVER
+
+const OSMElementData = props => {
+  const [selectedFeatureId, setSelectedFeatureId] = useState(null)
+  const [element, setElement] = useState(null)
+  const [fetchingElement, setFetchingElement] = useState(null)
+  const [fetchedElement, setFetchedElement] = useState(null)
```

Comment:
I cannot re-build the element id from its properties. As the JSON conversion remove the tag name. So I have its id but not its type.

Reply:
Not tracking it anymore, react is responsible for this